paymentsheet/src/test/java/com/stripe/android/paymentsheet/flowcontroller/DefaultFlowControllerTest.kt

```diff
@@ -1948,8 +1949,10 @@ internal class DefaultFlowControllerTest {
     }
 
     @Test
-    fun `On confirm existing payment method & PI, should send expected params to interceptor`() = testScope.runTest {
-        val flowController = createFlowController()
+    fun `On confirm existing payment method & PI, should send expected params to interceptor`() = runTest {
+        val flowController = createFlowController(
+            viewModelScope = CoroutineScope(UnconfinedTestDispatcher())
+        )
```

Comment:
Needed this since `collectLatest` is a permanent running coroutine until the scope is closed.